searchlib/src/vespa/searchlib/attribute/enumstore.h

```diff
@@ -168,11 +168,11 @@ class EnumStoreT : public IEnumStore {
             auto& entry = _store.get_entry_base(idx);
             entry.dec_ref_count();
             if (entry.get_ref_count() == 0) {
-                _possibly_unused.insert(idx);
+                _possibly_unused.push_back(idx);
             }
         }
         void commit() {
-            _store.free_unused_values(_possibly_unused);
+            _store.free_unused_values(std::move(_possibly_unused));
```

Comment:
Might need to sort _possibly_unused here, to avoid datastore freelist corruption (multiple entries with same ref).

Reply:
There is sorting in EnumStoreT<EntryT>::free_unused_values(IndexList to_remove).

Reply:
OK.